cfme/common/vm.py

```diff
@@ -531,6 +531,7 @@ def create(self, vm_name, provider, form_values=None, cancel=False, check_existi
                              handle_exception=True, num_sec=600)
                 elif override:
                     logger.info('Overriding exception to check failure condition.')
+                    return vm
```

Comment:
Question: For which provider you are getting `Exception` error? I am suspect about returning `vm` object in `override`.

Reply:
I am getting Exception error for **openstack** provider.

Reply:
Required: 
Then no need to return `vm` object `override` will handle an exception. As per your test no need of `vm object`. debug your failures.